treeFormatting.js

```diff
@@ -5,10 +5,10 @@ var categoryBrackets = {
 	"xp": "[]",
 	"phi": "()",
 	"x0": ["[x0 ","]"],
-	"w": ["(w ", ")"],
+	"w": ["[", "]"],
 	"clitic": ["",""],
 	"syll": ["",""],
-	"Ft": ["", ""],
+	"Ft": ["(", ")"],
```

Comment:
Let's make this by default ["(F", ")"]. I hadn't realized there were no foot brackets defined previously!